server/controllers/system/edit-settings.php

```diff
@@ -53,9 +53,13 @@ public function handler() {
             'max-size',
             'title',
             'url',
-            'mail-template-header-image'
+            'mail-template-header-image',
+            'default-is-locked',
+            'default-department-id'
         ];
 
+        $this->IsDefaultDepartmentValid();
```

Comment:
same as above, this name should be `check`, not `is`; it doesn't return anything